css/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: poppins;
+  src: url("https://fonts.googleapis.com/css?family=Poppins");
+}
+
+.header {
+  width: 375px;
+  height: 90px;
+  color: #6070ff;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+h2 {
+  font-size: 25px;
+}
+
```

Comment:
- [ ] I think it would be better to use the recommended styles given in the Figma template. What do you think about using `font-size: 18px;` for logo, which is same as present in the Figma template? 🤔🤓
- [ ] Also, please make sure to use and apply the correct `font-weight` as well which is `700`